### ISSUE TITLE
Unicode symbols in pages header not supported

### ISSUE THREAD

**Issue by VasiliyD** [wkhtmltopdf/wkhtmltopdf#3883]
Hi,
wkHtmlToPdf fails to display unicode symbols in pages header

If I run next command, the "русский текст" will be replaced with "????? ????" in header.
> wkhtmltopdf.exe --header-center "русский текст" Input.html Output.pdf

**Comment by hakimio** [comment]
Maybe just use html header instead?

**Comment by VasiliyD** [comment]
Yeah, that's definitely the first thing which I'm going to check as workaround. But it would be cool to fix the way how this tool parse unicode parameters (the root problem).
This will solve all sort of problems (like inability to convert files with unicode names, etc.)

**Comment by hakimio** [comment]
There hasn't been any significant activity on this project for quite a while. I doubt issue like yours will get any attention anytime soon unless you make a pull request yourself.

**Comment by VasiliyD** [comment]
I'm not sure I'll have time to look into it. But if there is some pitfalls or previous attempts to fix this, that somebody aware of, it would be helpful to know

**Comment by hakimio** [comment]
Maybe the following will be helpful:
https://github.com/wkhtmltopdf/wkhtmltopdf/pull/2858/commits/4b321bd5caab599f0c716c57e7b6005e9ad830a3
https://github.com/wkhtmltopdf/wkhtmltopdf/pull/2858

**Comment by mikehaertl** [comment]
I can not confirm this on Linux. I tested this with 0.12.5 and this command works just fine:

```
wkhtmltopdf --header-center "русский текст" index.html output.pdf
```

![wk_output_pdf](https://user-images.githubusercontent.com/675062/65405495-a3abce00-dddc-11e9-9f10-0b6aa2d64793.png)

Not though, that on Linux it's important that your locale env vars are set to a Unicode variant:

```
$ env | grep LANG
LANG=de_DE.UTF-8 
```

I have no idea how this is handled on Windows systems which is what the OP seems to be using.
